impl11/ddraw/SteamVR.cpp

```diff
@@ -354,16 +354,30 @@ void GetSteamVRPositionalData(float* yaw, float* pitch, float* roll, float* x, f
 			Matrix4 rotMatrixYaw, rotMatrixPitch, posMatrix;
 			rotMatrixYaw.identity();	rotMatrixYaw.rotateY(yaw_cockpitlook);
 			rotMatrixPitch.identity();	rotMatrixPitch.rotateX(-pitch_cockpitlook);
-			posMatrix.identity();		posMatrix.translate(-x_cockpitlook, -y_cockpitlook, z_cockpitlook);
+			posMatrix.identity();		posMatrix.translate(-x_cockpitlook, -y_cockpitlook, -z_cockpitlook);			
 			// Create inverse matrix by applying the opposite angles, in inverse order
 			m4_UndoCockpitLook = posMatrix * rotMatrixYaw * rotMatrixPitch;
 
-			// Invert it to get the transform from that pose into the head center coordinate reference,
-			// hopefully cancelling the pose estimation and the loss of precision (float to int32)
-			//mCockpitLook.invert();
+			Matrix4 m4_hmdPose = HmdMatrix34toMatrix4(trackedDevicePose.mDeviceToAbsoluteTracking);
+
+			// Recommended settings in CockpitLook.cfg to avoid jitter due to cockpit shake
+			// positional_x_multiplier = 0
+			// positional_y_multiplier = 0
+			// positional_z_multiplier = 0
+			// 
+			// DEBUG: you can cancel positional tracking completely by uncommenting these lines
+			// g_fPosXMultiplier = 0;
+			// g_fPosYMultiplier = 0;
+			// g_fPosZMultiplier = 0;
+
+			// Invert axes so that it works properly. I don't understand yet why this is necessary.
```

Comment:
I'm not sure either, but I do remember that the coordinate system in XWA is different from the coordinate system used by SteamVR. Maybe this sign inversion is caused by those differences.